src/pages/ErrorFallback.tsx

```
@@ -1,8 +1,28 @@
-import { FallbackProps } from 'react-error-boundary';
+import { useNavigate } from 'react-router-dom';
 
-export const ErrorFallback = ({ error }: FallbackProps) => (
-    <div>
-        <h2>Something Went Wrong: </h2>
-        <pre>{error.message}</pre>
-    </div>
-);
+import { Error } from '@components';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)